Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -21,6 +21,7 @@ @interface ViewController () <MZDayPickerDelegate, MZDayPickerDataSource>
 @property (nonatomic, strong) NSArray *allEvents;
 @property (nonatomic, strong) NSDictionary *eventsDictionary;
 @property (nonatomic, strong) NSArray *sortedDateKeys;
+@property (nonatomic, strong) NSDate *currentDateViewed;
```

Comment:
This variable name could be stronger. Perhaps something closer to `focusedDate` or something similar. There could be multiple dates being viewed, no?